### DISCUSSION TITLE
The new version of the Copilot becomes very slow or does not work

### DISCUSSION THREAD

**Discussion by garbe-github-support** [community/community#112150] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I use Copilot in PyCharm and Android Studio
Here is an example
This simple code even ran for a minute without giving a prompt. If it was in the past, it would immediately give a prompt
If your product is so poor, I am considering giving up my subscription and switching to products from other manufacturers

```
from .depth_anything_mandi import depth_anything_model_loader, Depth_Anything_Preprocessor_mandi
from .EfficienVitSamNode import GroundDinoProcessor, EfficientViTSamModelLoader, EfficientViTSamProcessor

NODE_CLASS_MAPPINGS = {
    "DepthAnythingModelLoader": depth_anything_model_loader,
    "DepthAnythingPreprocessor_mandi": Depth_Anything_Preprocessor_mandi,
    "GroundDinoProcessor": GroundDinoProcessor,
    # here need notice
}

NODE_DISPLAY_NAME_MAPPINGS = {
    "DepthAnythingModelLoader": "Depth Anything Model Loader",
    "DepthAnythingPreprocessor_mandi": "Depth Anything Preprocessor mandi"
}
```

**Comment by JeffreyBai** [comment]
C#/VS2022 too. Copilot suddenly stops generating useful suggestions. I think this problem has been around probably for a couple of days already and hasn't been taken care of yet.  It's really annoying and must be taken care of quickly.

**Comment by garbe-github-support** [comment]
It should be a proxy issue. After I turn off the VPN, it will be fine

**Comment by sudip358** [comment]
same here for me too :(
